docs/Templates.md

```diff
@@ -70,13 +70,13 @@ Platform templates define specific information about a platform that is used whe
     * `uid` (uint32, OPTIONAL). Override the uid and thus the user the container should run under.
     * `gid` (uint32, OPTIONAL). Override the gid and thus the user's group the container should run under.
     * `additionalGids` (array of uint32, OPTIONAL). Set the additional groups that should be assigned to the user running the container.
-* `logging`
+* `logging` (object, OPTIONAL)
   * `mode` (string, REQUIRED). Sets the logging sink for containers. Dobby by default supports the following logging modes:
     * `file` - log to a file
     * `journald` - log to journald
     * `devnull` - log to /dev/null (i.e. don't store container logs)
   * `logDir` (string, REQUIRED if `mode` is set to `file`). Directory to store container logs if logging to file
-* `dobby`
+* `dobby` (object, OPTIONAL)
   * `pluginDir` (string, REQUIRED). Location where Dobby plugins are found on the platform. Dobby plugins are by default found in `/usr/lib/plugins/dobby`
```

Comment:
Whilst we're here, can you change pluginDir and pluginDependencies here to OPTIONAL? That's what it's set to in the schema, and the correct value

Reply:
Done